### ISSUE TITLE
Subdep is performing an import

### ISSUE THREAD

**Issue by theogravity** [vinsonchuong/make-cert#10]
I'm not able to use the library because one of the sub-deps is using import syntax.

```
    /Users/theo/projects/datadog-send-log/node_modules/make-cert/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import forge from 'node-forge'
```

**Comment by theogravity** [comment]
The package needs to be transpiled to `commonjs`.

**Comment by vinsonchuong** [comment]
Hi! Thanks for trying my library!

So far, I've only needed to import this as an ES module. So, it doesn't yet export CJS.

I will update this to export CJS.

Could you please let me know the Node.js version you are using so I can ensure compatibility?

**Comment by theogravity** [comment]
v14, but I would probably do v10 to be safe

**Comment by vinsonchuong** [comment]
Cool! I'll find some time to work on it tonight.

Since the library is so small, I'll probably just make it fully CJS. I'll test on Node.js 10 as well.

**Comment by vinsonchuong** [comment]
@theogravity I released `v1.2.0`. The code is now exported as CJS. I tested the package in Node 10 and 14 as well.

Let me know if it works for you!
